### DISCUSSION TITLE
Can't compile any tools (specifically dark.exe)

### DISCUSSION THREAD

**Discussion by omriinbar-cyesec** [wixtoolset/issues#7818] [WiX Development]
**Encountered at:** wixtoolset/wix3

**Detail:**
I'm trying to compile dark.exe but using visual studio (tried 2019 and 2022) but no matter what I do, I keep getting this error:
```Strong name validation for the WiX toolset must be disabled on a developer machine to build. Execute 'msbuild C:\Users\user\source\repos\wix32\tools\OneTimeWixBuildInitialization.proj' from an elevated command prompt.```
does anyone know how to fix this?
thank you.

**Comment by robmen** [comment]
First, you need to msbuild `tools/OneTimeWixBuildInitialization.proj` elevated. Then, you'll want to build the `wix.proj` to get everything built once. Then you can iterate on a single tool.

**Reply by ijuedt** [reply]
tools/OneTimeWixBuildInitialization.proj: Can't find that in the repo. Has it been renamed?

**Reply by chrpai** [reply]
@ijuedt are you looking in the wix3 repos?  I see it at https://github.com/wixtoolset/wix3/blob/develop/tools/OneTimeWixBuildInitialization.proj

**Reply by ijuedt** [reply]
@chrpai My fault not seeing I'm in a wix 3 thread here. I've got wix 4 issues.
